dali/operators/reader/loader/video/frames_decoder_test.cc

```diff
@@ -21,11 +21,11 @@
 
 
 namespace dali {
-class FramesDecoderTest : public VideoTestBase {
+class FramesDecoderTest_CpuOnlyTests : public VideoTestBase {
```

Comment:
I think gtest is against putting `_` in test names. 

https://github.com/google/googletest/blob/main/docs/faq.md#why-should-test-suite-names-and-test-names-not-contain-underscore

Also, why is this name change necessary?

Reply:
We use this sufix to pick up tests we want to run in the CPU only test. This is already existing mechanism, so I just used it. 

Possible change in that to comply to gtest recommendation is out of scope for this, I think.

Reply:
Here is the test run: https://github.com/NVIDIA/DALI/blob/main/qa/TL0_cpu_only/test.sh#L34

Reply:
And here is how we use it in other tests: https://github.com/NVIDIA/DALI/blob/main/dali/kernels/slice/slice_cpu_test.cc#L75

Reply:
I think the main idea of the slice test here was to check CPU implementation than to check if it can run without GPU. We have only one such native test with that intention - CApiTest.